docs/evm-sidechain/xrpl-evm-sidechain-faq.md

```diff
@@ -0,0 +1,104 @@
+
+
+### What are the main benefits of the EVM sidechain?
+
+The main benefits are:
+
+- Faster time to launch by deploying applications as smart contracts.
+- No amendment approval needed to launch applications.
+- Make minor changes to existing smart contracts on other blockchains and launch them on the EVM sidechain.
+- Grow the XRPL ecosystem by adding EVM ecosystem projects and developers.
+
+
+### What are the main risks with the EVM sidechain?
+
+With the XRPL, developers use existing primitives such as payments, DEX, escrow etc. which are secure and have a great track record. With EVM applications, the security is the responsibility of the smart contract developer. Smart contract bugs can get exploited and they have led to a loss of funds.
+
+
+### What is the main use case for the EVM sidechain?
+
+From an XRPL community perspective, the EVM sidechain enables EVM developers to launch their existing applications in the XRPL ecosystem. Developers can now write smart contracts and build applications, and not be constrained by the capabilities of XRPL smart transactors. XRPL developers can launch applications that previously required XRPL protocol change and an amendment proposal which is complex, resource intensive, and risky.
+
+
+### How many EVM sidechains will there be? Can I set up my own?
+
+There will be one public EVM sidechain supported by Ripple and PeerSyst for the XRPL community. This EVM sidechain will have multiple validators run by XRPL community members. The software needed to run an EVM sidechain will be open source and any community member can use it to set up their own EVM sidechains, for permissioned use cases for example.
+
+
+### What are any critical dependencies to the launch of the EVM sidechain?
+
+The main dependency is that the amendment for cross-chain bridges (XLS-38d) proposal needs to be approved by the community and be live on the XRPL Mainnet. This is because the bridge between the XRPL and the EVM sidechain uses the XLS-38d bridge design.
+
+
+### Who will run the XRPL - EVM sidechain bridge?
+
+The bridge connecting the XRPL and the EVM sidechain will use the XLS-38d standard and will be operated by XRPL community members by running witness servers.
+
+
+### What does success look like for the EVM sidechain after launch on XRPL Mainnet?
+
+Long term, the EVM sidechain has a large and sustainable ecosystem of developers. The XRPL developer ecosystem has adopted the EVM sidechain as an extension of the XRPL and uses it as a testing ground before features are proposed on XRPL Mainnet or has deployed apps there that are not suitable for the XRPL Mainnet.
+
+
+### Who is our target user for the EVM sidechain?
+
+An EVM application developer who has experience writing Solidity applications and who is looking to monetize their applications on other EVM compatible blockchains outside of Ethereum. This developer isn’t happy about the high gas fees and low transaction speeds on Ethereum during peak usage.
```

Comment:
"This developer isn't happy about the high gas fees..."

This is reading like a blog post or marketing material, which I guess is the intent.